library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -88,11 +94,11 @@ class FsSnapshot
     #
     # @param number          [Fixnum]        Snapshot's number.
     # @param snapshot_type   [Symbol]        Snapshot's type: :pre, :post or :single.
-    # @param previous_number [Fixnum]        Previous snapshot's number.
-    # @param timestamp       [DateTime]      Timestamp
-    # @param user            [String]        Snapshot's owner username.
-    # @param cleanup_algo    [String]        Clean-up algorithm.
-    # @param description     [String]        Snapshot's description.
+    # @param previous_number [Fixnum, nil]   Previous snapshot's number.
+    # @param timestamp       [DateTime, nil] Timestamp
+    # @param user            [String, nil]   Snapshot's owner username.
+    # @param cleanup_algo    [Symbol, nil]   Clean-up algorithm.
+    # @param description     [String, nil]   Snapshot's description.
```

Comment:
please try to document what means if nil is passed as it is special case, so it deserve explanation.